### ISSUE TITLE
German 2-4-8 HIP 

### ISSUE THREAD

**Issue by mistertakai** [vasl-developers/vasl#58]
Art work is visible twice on HIP. Looks like the 468 is shining through.

![image](https://f.cloud.github.com/assets/3066447/2201431/d3e865da-98ee-11e3-8e26-81a10201f75e.png)

**Comment by JLeslie1004** [comment]
I will take a look at this. 

When I created the extension for the Dinant HASL (in playtest currently), I ran into the same issue because to create new 4-6-8/2-4-8 Assault Engineers for the module I started with the regular 4-6-8Elite/2-4-8Elite counters as a base. I solved the problem but at the moment I can't remember what I did to solve it. I'll figure it out again once I dig into it.

I also noticed that the German 2-3-8AE, 2-3-8Elite, 2-4-8Elite, 2-4-7, 2-3-7, and 2-3-6 HS's all do the squad ghost when HIP'd. I have not looked at other nationalities yet. 

In addition (and this probably needs to be a new issue), the 2-3-8AE HS Ctrl-V's to a 5-4-8Elite squad instead of a 5-4-8AE squad. If you take a 5-4-8AE squad and Ctrl-V it, it goes to a 2-3-8AE HS as expected, and the same HS you just Ctrl-V'd goes back to a 5-4-8AE squad as expected.

**Comment by BigAl737** [comment]
Been messing with this today and I don't see a cost effective fix. Replace with other is the best option I think but that is very time consuming and would likely induce other errors. The counter replacement system we have in place works very well for BH, ELR, broken, HS/SQ conversions... that this issue seems pretty minor to me. I'd suggest if anyone brings it up again we tell them to use the label option to define the counter. The label text should show clearly when HIP.

**Comment by derimmer** [comment]
I tested this on October 14, 2020 and the problem still exists exactly as shown in the original email. Low priority on my view but I will leave it on the issues list.

**Comment by BigAl737** [comment]
I agree Doug. I can look at this again at some point. Knowing a little more about VASSAL 4 years later there might be some other options:

You maybe could put a back ground colored layer between the icon layers that toggles on/off opposite the HIP activation.

You could make the HIP trait fully transparent and add a semi transparent white layer that toggles on and off with HIP activation.

I think either of those might work, more so the second option. It'd be a time consuming fix though as we're dealing with individual counters and not a prototype.

**Comment by derimmer** [comment]
Agree with you Al but let me put more stress on my view that this is low priority.

**Comment by BigAl737** [comment]
Got it, get it done right away. Understood sir :-)

**Comment by geezer09** [comment]
I have submitted a PR in VASSAL to fix this bug.(#13773).
